src/styles/main.scss

```diff
@@ -1,3 +1,212 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 300;
+  font-size: 12px;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+  font-weight: 500;
+
+  &__img {
+    margin: 10px 0;
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.nav {
+  height: 60px;
+  font-size: 12px;
+
+  &__link {
+    height: 60px;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    color: rgb(6, 11, 53);
+    margin: 0;
+    text-transform: uppercase;
+    padding: 0;
+  }
+
+  &__list {
+    padding: 0;
+    height: 100%;
+    display: flex;
+    margin: 0;
+    align-items: center;
+  }
+
+  &__item {
+    position: relative;
+    list-style: none;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin: 0;
+  }
+
+  &__link--is-active::after {
+    position: absolute;
+    background-color: #00acdc;
+    height: 4px;
+    width: 100%;
+    border-radius: 4px;
+    content: "";
+    display: block;
+    top: 56px;
+  }
+
+  &__link:hover,
+  &__link--is-active {
+    color: #00acdc;
+  }
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.wrap {
+  padding: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 944px;
```

Comment:
firstly, don't use width, use max-width instead, the layout should be adaptive

Reply:
secondly, you should calculate the width based on card width, gap and number of cards